src/components/governance/PageProposals.vue

```diff
@@ -1,56 +1,48 @@
 <template>
-  <PageContainer
```

Comment:
what was `PageContainer` doing?

Reply:
> what was `PageContainer` doing?

It was intended to replace TmPage and some of the internals we don't need there anymore.

I assume we will still need TmPage to be present?

Reply:
I think so true. I think it should be like:
```
<TmPage
  data-title="Proposals"
  :managed="false">
...
```

Reply:
Ok, I am going to change it